packages/c-popper/tests/use-popper.cy.tsx

```diff
@@ -0,0 +1,34 @@
+import * as Examples from '../examples'
+
+import { h, onMounted } from 'vue'
+import { usePopper } from '../src'
+import { useToggle } from '@vueuse/core'
+
+// Fails with isRef error
+xit('should work', () => {
+  cy.mount({
+    setup() {
+      const [isOpen, toggleIsOpen] = useToggle(true)
+
+      const { reference, referenceEl, popperEl, popper } = usePopper({
+        gutter: 16,
+        placement: 'right-end',
+      })
+
+      onMounted(() => {
+        _referenceEl = referenceEl.value
+        _popperEl = popperEl.value
+      })
```

Comment:
Looks like this `_referenceEl` and `_popperEl` are not defined. Could this be why the test is failing?

Reply:
Perhaps, I haven't looked at this file at all. @elevatebart has a ticket to fix it already.